library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html

```diff
@@ -223,7 +223,7 @@ <h4 class="mat-h4 margin-bottom-remove nav_sub_heading">Coming soon</h4>
   </ng-container>
 
   <ng-container *ngIf="hubs.hubname != 'Discuss' && !hubs.active">
-    <a class="flex flex-wrap mb-4 mobile" [id]="hubs.hubname" (click)="toggleVisibility();" [routerLink]="(hubs.path)" role="menuitem" i18n-aria-label>
+    <a class="flex flex-wrap mb-4 mobile" [id]="hubs.hubname" (click)="toggleVisibility(); trackTelemetry(hubs.hubname)" [routerLink]="(hubs.path)" role="menuitem" i18n-aria-label>
```

Comment:
Remove the semicolon else the build will fail.

Reply:
I've executed the prod build command locally without facing any problems, and we can use a semi-colon to separate function calls when necessary for multiple calls.